src/Sinch/Auth/ApplicationSignedAuth.cs

```diff
@@ -9,11 +9,11 @@ internal class ApplicationSignedAuth : ISinchAuth
     {
         private readonly string _appSecret;
         private readonly string _appKey;
-        private byte[] _jsonBodyInBytes;
-        private string _httpVerb;
-        private string _requestContentType;
-        private string _requestPath;
-        private string _timestamp;
+        private byte[]? _jsonBodyInBytes;
+        private string? _httpVerb;
+        private string? _requestContentType;
+        private string? _requestPath;
+        private string? _timestamp;
```

Comment:
Functionally wise, does it make sense to use nullable reference types for these parameters?

Reply:
Yes and no. They are not initialized with class so need to be nullable, I will to refactor this Auth class in later task.